### PR TITLE
Feature/split tiles tests

### PR DESCRIPTION
**Pull request by adrianmroz**

### CODE REVIEW

File: cypress/integration/split-tile.spec.js [hunk 1]
```diff
@@ -0,0 +1,432 @@
+/// <reference types="Cypress" />
+
+context("Split Tile", () => {
+  describe("No splits in View Definition", () => {
+    beforeEach(() => {
+      cy.visit("http://localhost:9090");
+    });
+
+    it("should load with no splits", () => {
+      cy.get(".split-tile .items")
```
Comment (bigsergey) [inline cypress/integration/split-tile.spec.js]: Move selectors to separate object:
```
const selectors = {
  splitItems: '.split-tile .items',
  addTile: '.split-tile .add-tile',
  ...
};
```

File: cypress/integration/split-tile.spec.js [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+        .should("have.length", 1)
+        .should("contain", "Channel");
+    });
+  });
+
+  describe("Channel split already in View Definition", () => {
+    beforeEach(() => {
+      // tslint:disable-next-line:max-line-length
+      cy.visit("http://localhost:9090/#wiki/4/N4IgbglgzgrghgGwgLzgFwgewHYgFwhpwBGCApiADSEQC2ZyOFBAomgMYD0AqgCoDCVEADMICNGQBOUfAG1QaAJ4AHZjXpDJZYfnUVqGegAUpWACYy88kGZiT0WXASMBGACJCoE5fgC0LgxU1BHQyLxAAXwBdaOooZSQ0S2slVV0vSQhsAHMhMzoybChHXXYACzhsbDIET0xJNHxQLR0CODMzMjMhVLUoUzC8iC12DBxdTqh2QvycoVVMzG6CSOokWghGvABWAAZYkH7Mwatm7SlC6d12zu7qYXradCbCIIntOBhxITBEGDVVq80gR6HBYFpIlFqMostUzG4CkVHMkQOVKtVatRsHB6PE4FdqNAAEqYYiYRpQkAwqpdADK9S2IBuXSE5GyM3w2C+CGoZQg2TKSAFWy5CAQESAA==");
+    });
```
Comment (bigsergey) [inline cypress/integration/split-tile.spec.js]: name this huge url

File: cypress/integration/split-tile.spec.js [hunk 3]
```diff
@@ -0,0 +1,432 @@
...
+      });
+    });
+
+    it("Channel dimension should not have Add split action", () => {
+      // Open Dimension Actions menu
+      cy.get('.dimension-list-tile .dimension .label:contains("Channel")').click();
+
+      cy.get(".dimension-actions-menu .subsplit.action").as("add-split");
+
+      cy.get("@add-split")
```
Comment (bigsergey) [inline cypress/integration/split-tile.spec.js]: what does mean `@add-split`? Is it kind of predefined action?

Comment (adrianmroz) [reply cypress/integration/split-tile.spec.js]: It is reference, defined line before with `as("add-split")`. Not sure if it is good practice, though.

File: cypress/integration/split-tile.spec.js (hunk 2 above)
Comment (adrianmroz) [reply cypress/integration/split-tile.spec.js]: I'll name them for now, but ideally I should write method that converts minimal `ViewDefinition` to url. That way you could see what to expect in UI. But that's story for another PR.

**Pull request merged**